app/controllers/search_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SearchController < ApplicationController
+
+  def index
+    @nearest_elec_station = StationFacade.get_nearest_station(params[:location])
+    start_address = params[:location]
+    end_address = @nearest_elec_station.first.full_address
```

Comment:
You're only ever displaying the first fuel station that you find, so you could let your `get_nearest_Station` method return just one fuel station object so you don't need to limit it in the controller.